src/server/bg_services/scrubber.js

```diff
@@ -73,7 +73,8 @@ async function build_chunks(req) {
     const chunk_ids = _.map(req.rpc_params.chunk_ids, id => MDStore.instance().make_md_id(id));
     const tier = req.rpc_params.tier && system_store.data.get_by_id(req.rpc_params.tier);
     const evict = req.rpc_params.evict;
-    const builder = new MapBuilder(chunk_ids, tier, evict);
+    const current_tiers = _.map(req.rpc_params.current_tiers, id => system_store.data.get_by_id(id));
```

Comment:
BTW a minor comment - lodash _.map is not really needed that much anymore, because there's a standard Array.map. Difference is that _.map also works on objects and nulls, but for arrays using the standard map function is enough. No need to go changing it all, but going forward we can try to use the standard js functions.

Reply:
Yeah, I did it only for consistency but now that I think, I could have made it consistent by changing the line above to use the native function 🤦 .

Reply:
Don't really make changes for this comment, I just wanted to note that for the future. Resolving.